### ISSUE TITLE
Update for Zotero 7

### ISSUE THREAD

**Issue by mjthoraval** [sharpevo/zotero-pdfkit#3]
Do you plan to update this plugin for Zotero 7?
The current version is not compatible with Zotero 7.

**Comment by sharpevo** [comment]
Probably. 

I'd like to say that the default pdf feature of this addon is essentially a trick, rather than an improvement that follows official design if you review the source code. Also, Zotero is planning to [make another attachment as default in a future version](https://forums.zotero.org/discussion/comment/436279/#Comment_436279). If it's not shipped with the new release, I will adapt it to Zotero 7.

**Comment by mjthoraval** [comment]
Indeed, it seems that a PR emerged in the Zotero GitHub after my post here:
https://github.com/zotero/zotero/pull/3333
Let's cross fingers it can make it to Zotero 7!

**Comment by crnkv** [comment]
A reminder that Zotero 7 has not yet implemented this feature officially as of the end of 2024

**Comment by crnkv** [comment]
I wrote a [script](https://github.com/crnkv/Zotero-Action-Scripts-Collection/blob/master/Set%20This%20as%20Default%20PDF.js) to be used in the [Zotero Actions and Tags](https://github.com/windingwind/zotero-actions-tags) to implement the date trick to set pdf as default.

**Comment by zfhxi** [comment]
> I wrote a [script](https://github.com/crnkv/Zotero-Action-Scripts-Collection/blob/master/Set%20This%20as%20Default%20PDF.js) to be used in the [Zotero Actions and Tags](https://github.com/windingwind/zotero-actions-tags) to implement the date trick to set pdf as default.

You, my friend, are the true hero!
